### PR TITLE
Issue #12: use downloaded latest cov build

### LINKED ISSUE #12: Make the crawler use the Firefox code coverage build downloaded by the latest_cov_build.py script

### PR DESCRIPTION
**Pull request by rhcu**
Fixes #12.

I have tried to use the latest downloaded coverage build binaries firefox-bin. 
Checked with Flake8. Seems to be working on local machine. 
I also deleted unnecessary taskcluster requirement and changed a little bit latest_cov_build.py
The last 4 commits are relevant to this issue. Previous were transferred because of conflict issue.

### CODE REVIEW

File: crawler.py [hunk 1]
```diff
@@ -179,6 +182,14 @@ def run_all(driver):
 
 os.environ['PATH'] += os.pathsep + os.path.abspath('tools')
 os.environ['MOZ_HEADLESS'] = '1'
-driver = webdriver.Firefox(firefox_binary='tools/nightly/firefox-bin')
+
+# Download the latest Firefox coverage build
+latest_cov_build()
```
Comment (marco-c) [inline crawler.py]: We probably shouldn't download it every time we run the crawler, but manually by running the script.

File: crawler.py [hunk 2]
```diff
@@ -179,6 +182,14 @@ def run_all(driver):
...
+
+# Webdriver uses Firefox Binaries from downloaded cov build
+my_path = os.path.abspath(os.path.dirname(__file__))
+filename = os.path.join(my_path, 'firefox/firefox-bin')
+binary = FirefoxBinary(filename)
+driver = webdriver.Firefox(firefox_binary=binary)
```
Comment (marco-c) [inline crawler.py]: I think you can simply do "firefox_binary=PATH_TO_BINARY"

File: crawler.py [hunk 3]
```diff
@@ -179,6 +179,10 @@ def run_all(driver):
 
 os.environ['PATH'] += os.pathsep + os.path.abspath('tools')
 os.environ['MOZ_HEADLESS'] = '1'
-driver = webdriver.Firefox(firefox_binary='tools/nightly/firefox-bin')
+
+# Webdriver uses Firefox Binaries from downloaded cov build
+my_path = os.path.abspath(os.path.dirname(__file__))
+filename = os.path.join(my_path, 'firefox/firefox-bin')
+driver = webdriver.Firefox(firefox_binary=filename)
```
Comment (marco-c) [inline crawler.py]: I think you can use the relative path here, no need to pass it an absolute path. Can you try if it works?

Comment (rhcu) [reply crawler.py]: I've tried it initially. It didn't work in that way; therefore, I've added an absolute path.

Comment (marco-c) [reply crawler.py]: This is strange, it works for me!

Comment (rhcu) [reply crawler.py]: Oops. Probably, it was before I have solved other problems. I can change it. :)

Comment (rhcu) [reply crawler.py]: @marco-c yep, it worked. :)

**Pull request merged**